src/ToggleButton/__tests__/__snapshots__/ToggleButton.test.js.snap

```diff
@@ -34,169 +34,17 @@ exports[`<ToggleButton /> should call componentDidUpdate and setState 1`] = `
   -webkit-transition: -webkit-transform 250ms ease;
   -webkit-transition: transform 250ms ease;
   transition: transform 250ms ease;
-  -webkit-transform: translateX(1.7rem);
-  -ms-transform: translateX(1.7rem);
-  transform: translateX(1.7rem);
+  -webkit-transform: translateX(0);
```

Comment:
Is that normal that some components are different ?

Reply:
@AntoninMarchardDev Can you be more explicit I don't understand?

Reply:
@spartDev `translateX` change from `translateX(1.7rem)` in the original snapshot to `translateX(0)` in the new snapshot is that normal ?